src/Client.js

```diff
@@ -1096,20 +1096,20 @@ class Client extends EventEmitter {
     }
 
     /**
-     * Returns the contact ID's profile picture URL, if privacy settings allow it
+     * Returns the chat ID's profile picture URL, if the client can view it
      * @param {string} contactId the whatsapp user's ID
-     * @returns {Promise<string>}
+     * @param {string} chatId the whatsapp chat's ID
      */
-    async getProfilePicUrl(contactId) {
-        const profilePic = await this.pupPage.evaluate(async contactId => {
+    async getPictureUrl(chatId) {
```

Comment:
```suggestion
    async getProfilePicUrl(chatId) {
```
I think the previous `getProfilePicUrl` method name will have more sence and be more user friendly than `getPictureUrl` (what picture do you mean here? )